Cargo.toml

```diff
@@ -17,7 +17,6 @@ features = ["approx", "mint", "rand"]
 
 [dependencies.nalgebra]
 version = "~0.18"
-default-features = false
```

Comment:
Is there any way to disable no std without enabling default features? I was doing this to reduce dependencies to try keep compile times down.

Reply:
std is the only default feature, but sure, calling it out explicitly works fine too.

Reply:
If std is the only default feature I'm happy for default features to just be enabled in this case.